sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -79,6 +79,9 @@ class SparkSession private(
 
   sparkContext.assertNotStopped()
 
+  /** The version of Spark on which this application is running. */
+  def version: String = SPARK_VERSION
```

Comment:
@rxin May I ask when should we use the `@Since` java annotation and when use `@since` in javadoc?